charms/knative-eventing/config.yaml

```diff
@@ -1,14 +1,8 @@
 # Copyright 2022 Canonical Ltd.
 # See LICENSE file for licensing details.
-#
-# TEMPLATE-TODO: change this example to suit your needs.
-# If you don't need a config, you can remove the file entirely.
-# It ties in to the example _on_config_changed handler in src/charm.py
-#
-# Learn more about config at: https://juju.is/docs/sdk/config
 
 options:
-  thing:
-    default: 🎁
-    description: A thing used by the charm.
+  eventing.namespace:
+    default: ""
+    description: The namespace to deploy Eventing to. If knative-serving is deployed in the same model, this namespace must be different to knative-serving's. This configuration is required.
```

Comment:
I got stuck a little on the wording.  Maybe something like:
`The namespace to deploy Eventing to.  This namespace cannot also contain another Knative Serving or Eventing instance.`?

Also, the value is required - is the default `""` valid?  If yes, then we should stay what it means here too (I guess it uses the model's namespace?)